shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1367,6 +1393,16 @@ - (void)startKeyBoardAnimation:(NSTimeInterval)duration {
       animations:^{
         // Set end value.
         [self keyboardAnimationView].frame = CGRectMake(0, self.targetViewInsetBottom, 0, 0);
+
+        // Set DisplayLink tracking values.
+        self.keyboardAnimationStartTime = fml::TimePoint().Now();
+        self.keyboardAnimationFrom = _viewportMetrics.physical_view_inset_bottom;
+        self.keyboardAnimationTo = self.targetViewInsetBottom;
```

Comment:
We can delete this `self.keyboardAnimationTo = self.targetViewInsetBottom;`
Also see https://github.com/flutter/engine/pull/37604/files#r1044120519 to delete redundant `keyboardAnimationTo `
The `targetViewInsetBottom` will be set before startKeyboardAnimation calls.

Reply:
sounds good!